ansible/roles/base/tasks/redhat_tasks.yml

```diff
@@ -30,14 +30,15 @@
 - name: install and start ntp
   shell: systemctl enable ntpd
 
-- name: install python-crypto
-  yum: name=python-crypto state=present
```

Comment:
Since the issue is with python-crypto installation I had added this task, which kind of allowed me have just one task for ansible installation with no conditionals and follow error-checking.

Note that if ansible installation were to fail for a reason other than python-crypto, the current change will cause the palybook to move on and not fail which is something we do not want.